src/abstractdataframe/abstractdataframe.jl

```diff
@@ -83,8 +83,13 @@ abstract type AbstractDataFrame end
 ##
 ##############################################################################
 
+"""
+    names(df::AbstractDataFrame)
+
+Return a vector of column names in the `df` data frame. The returned vector
+should not be modified. If you need to modify it use `copy(names(df))`.
+"""
```

Comment:
Also add docstring to the manual?

Reply:
added (I also proposed to switch Basics and Split-apply-combine sections as it is more natural to start with basics)